### PR TITLE
Make UK region filter a type ahead on companies collection

### PR DESCRIPTION
**Pull request by mforner13**
## Description of change

This pull request updates the UK Regions filter in the company's collection page to be a type ahead, rather than check boxes. 

The original uk region functional test (in companies) has been changed to test the new type ahead ability. 

## Test instructions

- Go to the companies collection page
- Filter UK Region by North West

_What should I see?_

### Before

Screenshot 2020-01-16 at 13 49 55

### After

Screenshot 2020-01-16 at 13 50 34

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: test/functional/cypress/specs/companies/filter-spec.js [hunk 1]
```diff
@@ -219,4 +205,24 @@ describe('Company Collections Filter', () => {
       )
     })
   })
+
+  it('should filter by uk region', () => {
+    const region = selectors.filter.UkRegion
+    const { typeahead } = selectors.filter
```
Comment (paulgain) [inline test/functional/cypress/specs/companies/filter-spec.js]: Looks good, one little thing: `const { UkRegion , typeahead } = selectors.filter`

Comment (mforner13) [reply test/functional/cypress/specs/companies/filter-spec.js]: Oo good thinking! Cheers Paul. I'm actually going to close this PR for now as i'll also be updating this feature in the other collection pages too and we want to release them all at once :) 

@ian-leggett for info

File: test/functional/cypress/specs/contacts/filter-spec.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 const selectors = require('../../../../selectors')
 
+console.log(selectors.filter.contacts.firstUkRegion)
```
Comment (ian-leggett) [inline test/functional/cypress/specs/contacts/filter-spec.js]: We need to remove this.

**Comment by paulgain** [comment]
I've just finished playing with this @mforner13 and it works really well. Good job!

**Pull request merged**